src/main/java/gov/cabinetofice/gapuserservice/dto/UserDto.java

```diff
@@ -15,6 +20,8 @@ public class UserDto {
     private List<Role> roles;
     private Role role;
     private Department department;
+    private Optional<Instant> created;
```

Comment:
Will this date be the moment the user is created? Do we instead want to set this value when the user runs through OL `/login`